js/ui/scheduler/ui.scheduler.appointments.layout_manager.js

```diff
@@ -75,19 +75,23 @@ class AppointmentLayoutManager {
         });
     }
 
-    _hasChangesInData(data) {
+    _isDataChanged(data) {
         const updatedData = this.instance.getUpdatedAppointment();
         return updatedData === data || this.instance.getUpdatedAppointmentKeys().some(item => data[item.key] === item.value);
     }
 
-    _hasChangesInSettings(settingList, oldSettingList) {
-        if(settingList.length !== oldSettingList.length) {
+    _isAppointmentShouldAppear(currentAppointment, sourceAppointment) {
```

Comment:
Sounds a bit strange when 'should' and 'is' are combined. Would it be better to use `shouldAppointmentAppear` instead?